javascript-1/script4.js

```diff
@@ -0,0 +1,6 @@
+console.log("EXERCITIUL 4");
+
+let nume="Lazaroiu";
+let prenume="Razvan Gabriel";
+
+console.log(`Numele meu este ${Lazaroiu} ${Razvan Gabriel}`);
```

Comment:
Aici trebuie sa folosesti variabila, pentur a afisa valoarea din el. In momentul de fata variabila Lazaroiu nu exista, exista doar nume si prenume, te rog sa modifici astfel incat sa foloseti variabilele. De asemenea te rog sa verifici daca exercitiile functioneaza, doarece codul din acest exercitiu o sa iti dea erori.